### ISSUE TITLE
ln: Set Displayed Units only effect on Lightning Network page.

### ISSUE THREAD

**Issue by 0x5826** [decred/decrediton#2391]

**Comment by matheusd** [comment]
I don't understand the issue. Can you be a bit more explicit?

**Comment by 0x5826** [comment]
I mean, We can add a feature that  display units " atoms" on Decrediton Lightning Network page and display on other pages units "DCR".

**Comment by matheusd** [comment]
Right, might be reasonable to do, though a bit of work given all our amount-related components get their unit from a single place.
